doc/make_doc.in

```diff
@@ -3,32 +3,57 @@
 set -e
 set -o pipefail
 
-abs_top_builddir=@abs_top_builddir@
-abs_top_srcdir=@abs_top_srcdir@
-
-GAP=$abs_top_builddir/bin/gap.sh
+GAP=@abs_top_builddir@/bin/gap.sh
 GAPARGS="-b -m 1g -x 80 -q -r --quitonbreak"
 
 echo "--------------------"
 echo "Building GAP manuals"
 echo "--------------------"
 
 "$GAP" $GAPARGS -A <<EOF
-if LoadPackage("io") = fail then
-  Error("Could not load IO package");
-fi;
-base:="$abs_top_srcdir";;
-books:=["ref", "tut", "changes", "hpc"];;
+base:="@abs_top_srcdir@";;
+books:=["ref", "tut", "changes", "hpc", "dev"];;
```

Comment:
I don't think we are going to include doc/dev manual in GAP 4.9, so I suggest that one should not attempt to build it if the directory doc/dev is not there. This will ensure that we are able to build manuals during release wrapping.